src/main/kotlin/me/serce/solidity/ide/hints/SolDocumentationProvider.kt

```diff
@@ -55,7 +55,7 @@ class SolDocumentationProvider : AbstractDocumentationProvider() {
     return builder.toString()
   }
 
-  private val keywordColors = SolHighlighter.keywords().plus(SolHighlighter.types()).filterNot { it == SolidityTokenTypes.RETURN }.map { it.debugName }
+  private val keywordColors = SolHighlighter.keywords().plus(SolHighlighter.types()).filterNot { it == SolidityTokenTypes.RETURN }.map { it.toString()   }
```

Comment:
cc @mbolotov, I replaced the usage following the guide from https://plugins.jetbrains.com/docs/intellij/api-internal.html#intellij-platform
<img width="935" alt="Screenshot 2023-04-02 at 1 51 15 pm" src="https://user-images.githubusercontent.com/1780970/229330357-11f2e783-e21f-49af-8ca1-5a8575b4bb20.png">